index.js

```diff
@@ -70,8 +61,14 @@ module.exports = (input, opts) => {
 				task: () => del('node_modules')
 			},
 			{
-				title: `Installing dependencies${opts.yarn ? ' using Yarn' : ''}`,
-				task: install(bin, opts.yarn)
+				title: 'Installing dependencies using Yarn',
+				enabled: () => opts.yarn === true,
+				task: () => exec('yarn', ['install'])
```

Comment:
>You can easily check for ENOENT

>If I have a dependency I cannot install, for instance a OSX only module on the Windows platform, it also triggers a fail-over. execa gives an ENOENT code whether the command could not be found or the installation failed (on Windows at least).

To your second point, yes, I am well aware how a yarn file works. My main issue was it can affect the local directory (and even np tests themselves). A user could be unaware that it defaulted to npm or doesn't understand the consequences of such. I know this type of issue wouldn't pop up often, but it is still a very real issue that could lead to a hard to diagnose problem.